README.md

```diff
@@ -77,3 +77,118 @@ pnpm run dev
 ## Front End Application Styling
 
 The `web` workspace, which houses the projects front end application uses SASS as a CSS pre-compiler. Style sheets should be post-fixed with a `.scss` file extension. `create-react-app` handles SCSS out-of-the-box so a loader is not necessary.
+
+# Front-End / Client
+
+## React Redux-Toolkit
+
+### navigate to apps/web and install
+
+`pnpm i @types/react-redux react-redux @reduxjs/toolkit --filter web`
+
+### create a store
+
+We create a store that holds our applications state, for now our store will only hold our application slice.
+
+```jsx
+import { configureStore } from '@reduxjs/toolkit'
+export const store = configureStore({
+  reducer: {
+    application: applicationReducer,
+  },
+})
+```
+
+After creating our store we need to wrap our entire app component with the redux provider
+
+```jsx
+<Provider store={store}>
+  <App />
+</Provider>
+```
+
+### Creating a Slice
+
+A slice is a collection of reducer logic, and actions that hold state specific to that slice. We only application for now.
+
+### typing our state with an interface
+
+```jsx
+interface CounterState {
+  value: number;
+}
+
+const initialState: CounterState = {
+  value: 0,
+}
+```
+
+### creating our reducer
+
+```jsx
+export const applicationSlice = createSlice({
+  name: 'application',
+  initialState,
+  reducers: {
+    updateIsTrue: (state, action: PayloadAction<boolean>) => {
+      state.isTrue = action.payload
+    },
+  },
+})
+```
+
+### exporting our actions, state, and reducer
+
+```jsx
+export const { updateIsTrue } = applicationSlice.actions
+export const selectApplication = (state: RootState) => state.application.isTrue
+export default applicationSlice.reducer
+```
+
+## installing Storybook
+
+### scaffold Storybook and install its dependencies
+
+add the following to the `.npmrc` at the root of your monorepo:
+
+`legacy-peer-deps=true`
+`node-linker=hoisted`
+
+next, initialized storybook, and install dev dependencies
+`pnpx sb init --skip-install`
+`pnpm install --save-dev @storybook/cli`
+
+### storybook-static
+
+To ensure build caching, you'll first need to add `storybook-static` to your `.gitignore`. Then, add `storybook-static` to the outputs of your `turbo.json` build task:
+
+```jsx
+"outputs": [
+"dist/**",
+"storybook-static/**"
+]
+```
+
+### running storybook & localhost simultaneously
+
+when adding a `dev` script to our root folder, turbopro finds all matching scripts and runs them.
+
+### handling 'modulenotfound error'
+
+we need to add this to the end of our `package.json` within our apps/web `package.json`. after adding, restart vscode
+
+```jsx
+"pnpm": {
+"overrides": {
+"enhanced-resolve":  "5.10.0"
+}}
+```
+
+### fixing ajv type-error
+
+we need to install two dependencies:
+`pnpm i schema-utils`
+
+### running storybook & localhost:
+
+`pnpm run start`
```

Comment:
This can be paired down significantly. I like the level of detail for like a docs wiki, but its a bit too much for the `readme`. Perhaps lose most of the technical installation and setup instruction and just focus on what other people would need to know to use our Redux system. Maybe a reference to how we organize - `slices` , `Ducks` pattern. Maybe point out the use of the custom typed hooks?